### ISSUE TITLE
Keyboard Cursor Should Follow Mouse Cursor

### ISSUE THREAD

**Issue by Elshara** [nvaccess/nvda#5974]
In NVDA, some documents can only be accessed via clicking the mouse. However, given NVDA's support of mouse text, there should be an option under mouse settings for the mouse cursor to follow the keyboard cursor. That way the enter key would function as a left click, and the mouse can be manipulated by navigating with normal keyboard commands. Additionally, for right click, NVDA enter can be used for quick access to the context and properties menu. Otherwise, number pad enter key would do well for this.

**Comment by jcsteh** [comment]
In browse mode, enter already clicks if appropriate. Mapping numpadEnter to right click is somewhat counter-intuitive and is also problematic for users without a numpad. NVDA+enter/numpadEnter is already used to activate the point beneath the review cursor. If you just want the mouse cursor to follow the review cursor, see #788.

**Comment by Elshara** [comment]
The mouse cursor I'm talking about, is the actual cursor touch pads
and USB mice use. I would like to be able to have a keyboard
accessible way of using the mouse remotely to click and drag as well
as review text only the mouse can see on the page. For example, try
using the keyboard to use the following demo site.
https://demo.socialengine.com/admin/?_ga=1.210121360.469478827.1462800326
When you login, you can see that there are links to options. But the
only way to access the drop down menu for those options to appear, is
by positioning the mouse. Similarly, click and drag interfaces for
moving modules around require the mouse for this feature. This is why
I request a way for the keyboard to physically act as a mouse cursor
for mouse commands.
